testing/Dockerfile.busted-buster

```
@@ -0,0 +1,3 @@
+FROM debian:buster
+RUN apt-get update -q && apt-get install -yq python3-pip python3-setuptools
+RUN echo -n "[easy_install]\nallow_hosts=localhost\nfind_links=/dist" > /root/.pydistutils.cfg
```

this is incorrect as it prints a single line:

```
$ echo -n "[easy_install]\nallow_hosts=localhost\nfind_links=/dist"
[easy_install]\nallow_hosts=localhost\nfind_links=/dist
```

`echo -e` could work, or better yet: use `printf` to move _all_ the escapes into the string (instead of having the intermediate newlines added by the escapes, and the final newline implicitly by `echo`)

```suggestion
RUN printf "[easy_install]\nallow_hosts=localhost\nfind_links=/dist\n" > /root/.pydistutils.cfg
```